wintermotes/week5/ass6/app/components/fields.js

```diff
@@ -0,0 +1,220 @@
+var React = require('react');
+var List = require('./list');
+var Messages = require('./message').Messages;
+var Message = require('./message').Message; 
+var archivedMessages = require('./message').ArchivedMessages;
+var ArchivedMessage = require('./message').ArchivedMessage;
+
+var ListStore = require('../stores/list-store');
+var ListActions = require('../actions/list-actions');
+
+var MessageStore = require('../stores/message-store');
+var MessageActions = require('../actions/message-actions');
+
+import NotificationBar from '../components/notification-bar'
+
+function getLists(){
+	return ListStore.getAllLists();
+}
+
+function getNotifications(){
+	return MessageStore.getNotifications()
+}
+var CreateListField = React.createClass({
```

Comment:
So how would you name it then? For instance, if i had several list fields, one might edit the list, another might delete the list etc. I do the same name conventions with fields manipulating the message components as well, and wanted to stay consistent.

I'm not at all saying this is the right approach, but it could be great if i had a counterexample. :)